script/release/01.push-release-branches-DSL.txt

```diff
@@ -0,0 +1,66 @@
+def cloneScript_01 = 
```

Comment:
Another possibility is to do the following: 

- add a properties file in this directory (on your laptop) that contains these values
- add a .gitignore file that ignores the properties file
- add code to the script that reads in the properties file and assigns/uses the values..